jamon-parent/jamon/src/main/java/org/wicketstuff/jamon/monitor/JamonRepository.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wicketstuff.jamon.monitor;
+
+import static java.lang.String.format;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import com.jamonapi.Monitor;
+import com.jamonapi.MonitorFactory;
+
+/**
+ * <p>
+ * Repository around the Jamon API. This repository is a small wrapper around the
+ * {@link MonitorFactory}.
+ * </p>
+ * <p>
+ * Since Wicket is an unmanaged framework this repository is implemented as a singleton. All
+ * monitors are kept in memory anyway so that shouldn't pose a problem. One can use the
+ * {@link #clear()} method to remove all {@link Monitor}s from this repository.
+ * </p>
+ * 
+ * @author lars
+ * 
+ */
+@SuppressWarnings("serial")
+public class JamonRepository implements Serializable
+{
+	private static final JamonRepository jamonRepository = new JamonRepository();
```

Comment:
This could be stored in Application's metadata instead of being a `static` field.